TextToSpeech/impl/TTSManager.cpp

```diff
@@ -114,9 +114,33 @@ TTS_Error TTSManager::listVoices(std::string language, std::vector<std::string>
     return TTS_OK;
 }
 
+TTS_Error TTSManager::listLocalVoices(std::string language, std::vector<std::string> &voices) {
+     
+     std::string key = std::string("voice_for_");
+     if(language.empty())
+     {
+        voices.push_back(m_defaultConfiguration.localvoice());
+     }
+     else
+     {
+         key += language;
```

Comment:
the key doesn't seem to be used above. perhaps you can move it here and use a const to denote that it doesn't change:

const std::string key = std::string("voice_for_") + language;